src/acl/external/LDAP_group/required.m4

```diff
@@ -5,4 +5,6 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_CHECK_HEADERS([ldap.h winldap.h],[BUILD_HELPER="LDAP_group"])
+AS_IF([test "x$squid_cv_LDAP" != "x"],[
+    AC_CHECK_HEADERS([ldap.h winldap.h],[BUILD_HELPER="LDAP_group"])
```

Comment:
Can we adjust SQUID_CHECK_LIB_WORKS(ldap) so that we never have to check these headers in post-SQUID_CHECK_LIB_WORKS(ldap) code (when LDAP support has been detected)?

Reply:
Even better, we can decouple detecting them from pkg-config